app: nrfcloud: Add GCI support to cellpos (#270)
nrfconnect/ncs-serial-modem: doc/app/at_nrfcloud.rst

::

   AT#XNRFCLOUDPOS=<cell_count>,<wifi_pos>[,<MAC 1>[,<RSSI 1>],<MAC 2>[,<RSSI 2>][,<MAC 3>[...]]]

* The ``<cell_count>`` parameter indicates the number of cells to include in the location request.

>   The value range is ``0`` to ``15``. A good suggested value for cellular positioning is ``4``.

Reviewer: ```suggestion
  The value range is ``0`` to ``15``. 
  For cellular positioning, a recommended value is ``4``.
```

  ``0`` means that no cellular network information will be included in the location request.

>   The |SM| uses the ``AT%NCELLMEAS`` command to retrieve the cellular network information, and depending on the value of ``<cell_count>``, it may use the command multiple times.

Reviewer: ```suggestion
  The |SM| uses the ``AT%NCELLMEAS`` command to retrieve the cellular network information, and depending on the value of ``<cell_count>``, the command might be executed multiple times.
```
